publish_hw_status_frequency:             2.0
publish_software_version_frequency:      2.0
publish_learning_mode_frequency:         2.0